site/docs/tasks/using-opa.md

```diff
@@ -25,6 +26,14 @@ You can also add conditions like
 
 The verdict `allow` will be reached only if the conditions hold, and no other rule has been triggered, e.g. a rule requiring column redaction.
 
+## Input to policies
+
+The input object includes the application properties and the requested action as well as dataset details (id, metadata).
+
+- `properties`: application/workload properties defined in FybrikApplication, e.g. `properties.intent`
```

Comment:
I think it was `context` instead of `properties`

Reply:
Fixed. Thanks